modules/aioseop_sitemap.php

```diff
@@ -4636,6 +4623,56 @@ private function get_prio_freq_static_homepage( $links ) {
 			return $links;
 		}
 
+		/**
+		 * Sets the priority and frequency for the static blogpage.
+		 *
+		 * @since   3.4.0
+		 *
+		 * @param   array   $links
+		 * @return  array   $links
+		 */
+		private function get_prio_freq_static_blogpage( $links ) {
+			$blogpage_id = (int) get_option( 'page_for_posts' );
+			$permalink   = get_permalink( $blogpage_id );
+
+			if ( 0 === $blogpage_id || 'page' !== get_option( 'show_on_front' ) ) {
+				return $links;
+			}
+
+			$blogpage_index = array_search( $permalink, array_column( $links, 'loc' ) ); // phpcs:ignore PHPCompatibility.FunctionUse.NewFunctions.array_columnFound
```

Comment:
I am pretty sure array_column is a PHP 5.5+ feature. We will need to test with our minimum supported version, but I think we should just swap it out.

Reply:
@benjaminprojas correct, but we have a compatibility function for this - https://github.com/awesomemotive/all-in-one-seo-pack/blob/caecc441411a4812f036afe0df72f70668c0b430/inc/compatibility/php-functions.php#L8-L25

Reply:
ok cool